plugins/legacy-jbrowse/package.json

```
@@ -44,5 +44,5 @@
     "mobx-state-tree": "3.14.1",
     "rxjs": "^6.0.0"
   },
-  "private": true
+  "private": false
```

To match our other published packages, I think we should remove the "private" field and replace it with the publishConfig:

```suggestion
  "publishConfig": {
    "access": "public"
  }
```